File: DSA/Madhav_590014266/Day3/Q2.java
public class NumberOfOccurrence {

    public static int countOccurrences(int[] arr, int target) {
        int count = 0;

        for (int num : arr) {
            if (num == target) {
                count++;
            }
        }

        return count;
    }

    public static void main(String[] args) {
        int[] arr = {1, 1, 2, 2, 2, 2};
        int target = 2;

        int result = countOccurrences(arr, target);
        System.out.println("Number of occurrences: " + result);
    }
}

File: DSA/Prarthana_Baisoya_590024838/Day_4/q2.c
#include <stdio.h>

int main()
{
    int n;

    scanf("%d", &n);

    int arr[100];
    int result[100];

    for(int i = 0; i < n; i++)
    {
        scanf("%d", &arr[i]);
    }

    int k;
    scanf("%d", &k);

    k = k % n;

    for(int i = 0; i < n; i++)
    {
        result[(i + k) % n] = arr[i];
    }

    for(int i = 0; i < n; i++)
    {
        printf("%d ", result[i]);
    }

    return 0;
}
